src/view/input/cli_input.rs

```diff
@@ -14,13 +14,16 @@ pub struct CliInput {
 pub enum CliCommandChoice {
     DefaultBasicRequest {
         url: String,
+        print_body_only: bool,
```

Comment:
For better extensibility put this "print_body_only" field in CliInput with a "view_options". Like this
```rs
pub struct CliInput {
    pub choice: CliCommandChoice,
    pub request_input: RequestBuildingOptions,
    pub save_options: SavingOptions,
    pub view_options: ViewOptions,
}

pub struct ViewOptions {
    pub print_body_only: bool
}
```
For this ViewOptions, follow the pattern for RequestBuildingOptions and SavingOptions: using `from_clap_matches` function to build it from clap matcher

With this, if we need to create a flag for headers, quiet, verbose, whatever it's easier. Would not be necessary change ever enum value declaration. Only  the function for build it.